### ISSUE TITLE
when installing magisk its not installing.

### ISSUE THREAD

**Issue by SuperErnD** [casualsnek/waydroid_script#137]
![image](https://github.com/casualsnek/waydroid_script/assets/84639576/dbedf52a-d061-4b70-b343-729faa2c63bc)
when installing magisk its not installing. script dont give any errors 
linux kernel - 6.6.2-cb2.0.fc39.x86_64
distro - fedora 39 workstation

**Comment by dvoracekma2019** [comment]
Same here on archlinux with linux-zen kernel. Tried also with linux-xanmod-anbox kernel, that didn't work either.

**Comment by casualsnek** [comment]
Click on the install button on the magisk app.
    1. On the subsequent screen, select the option “Direct install into system partition.“
    4. Click on the “Let’s go” button located next to the selected option.
    5. When it’s done, click the “Restart” button shown at the bottom right.
If this does not work as well let me know !

**Comment by dvoracekma2019** [comment]
Hi, thanks for you reply ^^
Howewer these options aren't here at all, It seems like It completle failed to root It. I don't know If I'm doing something wrong.
Tried already on Debian, Fedora and Arch, while distro hoping.

**Comment by 0a00** [comment]
I don't have this option in my environment either,
![图片](https://github.com/casualsnek/waydroid_script/assets/59482058/5088ce2d-03cb-4333-9fd8-daef6a2b4094)

**Comment by iFlxy** [comment]
I'm having the same issue.

**Comment by ryanrudolfoba** [comment]
Same behavior for me on Fedora install.

**Comment by blacklightpy** [comment]
Same here on Void Linux

**Comment by mistrmochov** [comment]
Unfortunately It doesn't work on any liznux distribution.

**Comment by ryanrudolfoba** [comment]
> Unfortunately It doesn't work on any liznux distribution. You can go with my script temporarily https://github.com/mistrmochov/magiskWaydroid

i tried yours the other day and it didnt work either. gonna try again on an older lineageos build from Nov / early Dec

**Comment by casualsnek** [comment]
Thanks for reporting, latest magisk-delta has some issues starting on magisk. I will try downgrading magisk to get it working at the moment, till i find a way to get latest builds working again !

**Comment by casualsnek** [comment]
It's downgraded now ! Magisk should work now. Can you test it ?

**Comment by Techwizz-somboo** [comment]
> It's downgraded now ! Magisk should work now. Can you test it ?

I just tried. Having the same problem as everyone else listed above.

**Comment by Techwizz-somboo** [comment]
> > > It's downgraded now ! Magisk should work now. Can you test it ?
> > 
> > 
> > I just tried. Having the same problem as everyone else listed above.
> 
> If you did a dirty downgrade (downgrade while you previously installed magisk.. You'll need to clean up some files for downgrade to work). I will share a write-up with instruction here after a while !

It was a brand new installation

**Comment by casualsnek** [comment]
> > > > It's downgraded now ! Magisk should work now. Can you test it ?
> > > 
> > > 
> > > I just tried. Having the same problem as everyone else listed above.
> > 
> > 
> > If you did a dirty downgrade (downgrade while you previously installed magisk.. You'll need to clean up some files for downgrade to work). I will share a write-up with instruction here after a while !
> 
> It was a brand new installation

Weird ! Can you share output of ``cat /proc/cmdline`` and ``uname -r`` ?

**Comment by Techwizz-somboo** [comment]
> > > > > It's downgraded now ! Magisk should work now. Can you test it ?
> > > > 
> > > > 
> > > > I just tried. Having the same problem as everyone else listed above.
> > > 
> > > 
> > > If you did a dirty downgrade (downgrade while you previously installed magisk.. You'll need to clean up some files for downgrade to work). I will share a write-up with instruction here after a while !
> > 
> > 
> > It was a brand new installation
> 
> Weird ! Can you share output of `cat /proc/cmdline` and `uname -r` ?

For me the solution was to swap to this kernel. So probably not a problem with your script https://aur.archlinux.org/packages/linux-xanmod-anbox

**Comment by casualsnek** [comment]
> > > > > > It's downgraded now ! Magisk should work now. Can you test it ?
> > > > > 
> > > > > 
> > > > > I just tried. Having the same problem as everyone else listed above.
> > > > 
> > > > 
> > > > If you did a dirty downgrade (downgrade while you previously installed magisk.. You'll need to clean up some files for downgrade to work). I will share a write-up with instruction here after a while !
> > > 
> > > 
> > > It was a brand new installation
> > 
> > 
> > Weird ! Can you share output of `cat /proc/cmdline` and `uname -r` ?
> 
> For me the solution was to swap to this kernel. So probably not a problem with your script https://aur.archlinux.org/packages/linux-xanmod-anbox

Yes, xanmod is must with psi=1 kernel parameter to get magisk running on waydroid on Arch !

**Comment by AlecBlance** [comment]
> I got Magisk installed and Zygisk enabled here: [#143 (comment)](https://github.com/casualsnek/waydroid_script/pull/143#issuecomment-1893679352)

Thanks! I got my Zygisk this way. For now, this will suffice but, it'd be good if we will have the latest magisk-delta. Hopefully, they will sort it out.
